ntbs-service/Pages/Notifications/Overview.cshtml.cs

```diff
@@ -22,22 +20,26 @@ public async Task<IActionResult> OnGetAsync(int id)
             }
 
             NotificationId = Notification.NotificationId;
+            await GetLinkedNotifications();

```

Comment:
I decided to still show the linked notification banner for notifications that don't have edit permission - I think that makes sense? If someone ends up here by clicking through from another linked notification, they should be able to easily go back